twistlock/datadog_checks/twistlock/twistlock.py

```diff
@@ -0,0 +1,322 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from collections import Counter
```

Comment:
Use `defaultdict(int)` instead for performance

Reply:
okay